core/sql/exp/exp_eval.cpp

```diff
@@ -862,7 +862,17 @@ ex_expr::exp_return_type ex_expr::evalClauses(ex_clause *clause,
 	atp1->setDiagsArea(diagsArea);
 
       if (retcode == ex_expr::EXPR_ERROR)
-	return retcode;
+      {
+        if((getPCodeMode() & ex_expr::ERROR_CONTINUE) > 0)
+        {
+          ExpTupleDesc::setNullValue( op_data[0],
+                                      clause->getOperand(0)->getNullBitIndex(),
+                                      clause->getOperand(0)->getTupleFormat() );
+          retcode = ex_expr::EXPR_OK;
```

Comment:
-- this code also will return back from line #871 on the first conversion clause error instead
   of continuing to the next clause.
-- it will also not work correctly if getPCodeMode() is not  set.